service/notifier.go

```diff
@@ -250,3 +262,29 @@ func (p *notifier) decodeSearchAttrBinary(bytes []byte, key string) interface{}
 	}
 	return val
 }
+
+func (p *notifier) sendMessageToWebhook(notification *Notification, webhook *config.Webhook) error {
+	var jsonStr = []byte(fmt.Sprintf("%v", notification))
+	req, err := http.NewRequest("POST", webhook.URL.String(), bytes.NewBuffer(jsonStr))
+	if err != nil {
+		return err
+	}
+	req.Header.Set("X-Custom-Header", "myvalue")
+	req.Header.Set("Content-Type", "application/json")
+
+	// TODO: setup retry logic using webhook config
```

Comment:
I created an issue to track this for next PR: https://github.com/cadence-oss/cadence-notification/issues/11